test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,25 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNullTest() throws Exception {
```

Comment:
You can add more test cases, such as empty `Objects`, or `Objects` that contain other data types like numbers, arrays, etc.